sub/sd_ass.c

```diff
@@ -288,46 +288,53 @@ static int init(struct sd *sd)
     return 0;
 }
 
+static inline bool is_escaped(const char *str, size_t pos)
```

Comment:
As discussed in the previous PR, this isn't how ASS bracket escaping works, and isn't necessary:
- A `{` is escaped iff it's preceded by a `\` (no matter how many `\`s there are)
- The `}` terminating a `{` block cannot be escaped

Reply:
This makes it a lot more straight forward, I missed exact details on escaping.

Reply:
Ok, removed unnecessary escaping checks.